client/package.json

```diff
@@ -54,7 +54,7 @@
                     "scope": "window",
                     "type": "string",
                     "default": "",
-                    "description": "Absolute path to Yseop CLI executable."
+                    "description": "Absolute path to Yseop CLI executable (which is in the bin/ directory where Yseop CLI was installed)."
```

Comment:
* Sounds like CLI is in a bin directory rather than the opposite.
* This does not mention the fact that there are two files in there (Win / Lin).

Reply:
I mostly wanna say “Absolute path to your Yseop CLI executable (see Yseop CLI doc for more info).”

Reply:
Wouldn't hurt to add

> (“yseop” or “yseop.bat”)

to that.